### PR TITLE
Kf1.7 volume table stuff

### PR DESCRIPTION
**Pull request by mathis-marcotte**
closes https://github.com/StatCan/aaw/issues/1706

### CODE REVIEW

File: frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts [hunk 1]
```diff
@@ -291,46 +303,38 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
...
-      if(mounts[element.name]){
-        element.usedBy = mounts[element.name].name;
-        element.status = {} as Status;
-        element.status.message = $localize`Attached`;
-        element.status.phase = STATUS_TYPE.MOUNTED;
-      } else {
-        element.status = {} as Status;
-        element.status.message = $localize`Unattached`;
-        element.status.phase = STATUS_TYPE.UNMOUNTED;
-      }
```
Comment (wg102) [inline frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts:315]: Attached vs unattached is ours

File: frontend/jupyter/src/app/pages/index/index-default/config.ts [hunk 2]
```diff
@@ -154,54 +157,20 @@ export const defaultConfig: TableConfig = {
...
-        tooltipField: 'name',
-        truncate: true,
-      }),
-    },
-    {
-      matHeaderCellDef: '',
-      matColumnDef: 'prot-b',
-      value: new ComponentValue({
-        component: ProtBComponent,
-      }),
```
[the remark this answers is not in the document]
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/pages/index/index-default/config.ts:179]: this table config got moved to index/config.ts, but yes i forgot to add back the pro b in there.

File: frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts (hunk 1 above)
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts:315]: yup. Thanks. Added that back in. With the update, there is a status field that should be coming from the backend. But i think in our custom case we just want attached or unattached.

File: frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts [hunk 3]
```diff
@@ -291,46 +303,38 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
...
-    });
+  public parseIncomingData(pvcs: PVCResponseObject[]): PVCProcessedObject[] {
+    const pvcsCopy = JSON.parse(JSON.stringify(pvcs)) as PVCProcessedObject[];
 
     for (const pvc of pvcsCopy) {
       pvc.deleteAction = this.parseDeletionActionStatus(pvc);
-      pvc.protB = this.parseProtBVolume(pvc);
```
[the remark this answers is not in the document]
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts:320]: yup. missed that. Added back in. Thanks!

**Pull request merged**